gtk2_ardour/editor.cc

```diff
@@ -4471,7 +4479,8 @@ Editor::VisualState::VisualState (bool with_tracks)
 
 Editor::VisualState::~VisualState ()
 {
-	delete gui_state;
+	if (gui_state)
```

Comment:
See https://ardour.org/styleguide.html item 25

> Don't write: `if (foo) delete foo;` ... C++ handles `delete (0);` just fine, so just `delete foo` will do.

Reply:
Ok, thanks you, learned something new.

Reply:
Do I need to make another pull request?